content/sensu-core/2.0/getting-started/installation-and-configuration.md

```diff
@@ -118,103 +122,257 @@ sudo apt-get install sensu-agent
 {{< platformBlock "RHEL/CentOS" >}}
 #### RHEL/CentOS
 
-Add the Sensu beta repository.
+Add the Sensu repository.
 
 {{< highlight shell >}}
 curl -s https://packagecloud.io/install/repositories/sensu/beta/script.rpm.sh | sudo bash
 {{< /highlight >}}
 
-Install the Sensu backend and agent packages.
+Install the sensu-agent package.
 
 {{< highlight shell >}}
 sudo yum install sensu-agent
 {{< /highlight >}}
+
 {{< platformBlockClose >}}
 
 {{< platformBlock "Windows" >}}
+
 #### Windows
 
 Download the [Sensu agent binary](https://storage.googleapis.com/sensu-binaries/2.0.0-beta.4-1/windows/amd64/sensu-agent).
+
 {{< platformBlockClose >}}
 
 {{< platformBlock "Ubuntu/Debian RHEL/CentOS Windows" >}}
 
 ### 2. Create the configuration file
 
+{{< platformBlockClose >}}
+
+{{< platformBlock "Ubuntu/Debian RHEL/CentOS" >}}
+
+#### Ubuntu/Debian/RHEL/CentOS
+
 Copy the example agent config file to the default config path.
 
 {{< highlight shell >}}
 sudo cp /etc/sensu/agent.yml.example /etc/sensu/agent.yml
 {{< /highlight >}}
 
-In order for the agent to function, it will need to have a list of one or more backends to point to. This can be set
-by setting `backend-url` to the ip and port of a Sensu backend.
+In order for the agent to function, it will need to have a list of one or more backends to point to.
+Open `/etc/sensu/agent.yml` and set `backend-url` to the IP and port of a Sensu backend.
 
 {{< highlight yaml >}}
 backend-url:
   - "ws://127.0.0.1:8081"
 {{< /highlight >}}
 
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+#### Windows
+
+Coming soon.
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Ubuntu/Debian RHEL/CentOS Windows" >}}
+
 ### 3. Start the service
 
 {{< platformBlockClose >}}
 
 {{< platformBlock "Ubuntu/Debian RHEL/CentOS" >}}
+
 #### Ubuntu/Debian/RHEL/CentOS
 
-Start the services using systemd.
+Start the agent using systemd.
 
 {{< highlight shell >}}
 sudo systemctl start sensu-agent
 {{< /highlight >}}
 
-You can verify that the services are properly running using the log files.
+You can verify that the agent is running properly using the log files.
```

Comment:
```suggestion
You can verify that sensu-agent is running properly using the log files.
```

Reply:
I'm thinking it may be better to say the service that we're verifying instead of just saying "agent"